osc-domain/src/main/java/org/osc/core/broker/model/entities/virtualization/SecurityGroup.java

```diff
@@ -174,8 +179,17 @@ public String getNetworkElementId() {
     public void setNetworkElementId(String networkElemId) {
         this.networkElementId = networkElemId;
     }
+    
 
-    @Override
+    public ServiceFunctionChain getServiceFunctionChain() {
+		return serviceFunctionChain;
+	}
```

Comment:
Please use spaces instead of tabs and ensure indentation is aligned. You can configure eclipse to do this automatically.

Reply:
It still has tabs